public/js/spec/interfaceSpec.js

```diff
@@ -0,0 +1,8 @@
+describe('Interface', () => {
+  describe('showHeadlinesAndPictures', () => {
+    it('renders the headlines and pictures', () => {
+      // frontPage.articles = [{headline: 'Test', thumb}]
+      // expect(headlines).toInclude('Test')
```

Comment:
what happened here? :)

Reply:
Wasn't sure at the time how to write an effective test for this method. I think maybe with a fresh mentality I might be able to come up with a solution now :)